paddle/fluid/platform/mkldnn_reuse.h

```diff
@@ -120,6 +120,15 @@ class MKLDNNHandlerT {
     return (dev_ctx_.GetBlob(key_p) != nullptr);
   }
 
+  bool isBwdCached() {
```

Comment:
Function name is misleading. It changes the bwd_pd_ for the object

Reply:
It is misleading. I added it so behaviour is the same (by analogy) as of isCached() . Once we carry on with reafactoring we will fix this (it is a bigger change ).